src/applications/common/schemaform/review/ReviewCollapsibleChapter.jsx

```diff
@@ -127,6 +127,9 @@ export default class ReviewCollapsibleChapter extends React.Component {
               pageData = form.data;
               fullPageKey = page.pageKey;
             }
+            if (page.hasCustomArrayField) {
```

Comment:
Is this necessary if you use the `keepInPageOnReview` option?

Reply:
Thanks @jbalboni, I had forgotten about that option, which displays the following:
<img width="622" alt="screen shot 2018-05-16 at 3 39 13 pm" src="https://user-images.githubusercontent.com/16051172/40147657-6872a7d4-591f-11e8-8f55-a27dfffa7ee9.png">
![image](https://user-images.githubusercontent.com/16051172/40147698-910336a0-591f-11e8-846d-ab945c598ec9.png)
Is it possible to configure a summary view? It seems like `reviewMode` is not available?
@goldenmeanie is a summary view of interest? Or is simply displaying the form the plan here?